e2e/atlas_dbusers_test.go

```diff
@@ -81,6 +81,33 @@ func TestAtlasDBUsers(t *testing.T) {
 		}
 	})
 
+	t.Run("Update", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			atlasEntity,
+			dbusersEntity,
+			"update",
+			username,
+			"--role",
+			"readWrite")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		}
+
+		user := mongodbatlas.DatabaseUser{}
+		err = json.Unmarshal(resp, &user)
+		if err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+
+		if user.Username != username {
```

Comment:
Thanks for adding an e2e test, could we add a check to see we actually changed the roles? I guess that's the only thing that would change from this API